core/modules/file/file.admin.inc

```diff
@@ -677,3 +677,41 @@ function _file_sort_weight_label($a, $b) {
     return $a_weight < $b_weight ? -1 : 1;
   }
 }
+
+/**
+ * Confirm form for file type classification process.
+ *
+ * @see file_type_classify_confirm_submit()
+ * @see file_menu()
+ *
+ * @ingroup forms
+ */
+function file_type_classify_confirm($form, $form_state) {
+  return confirm_form(array(), t('Are you sure you want to classify the file types?'),
+  'admin/reports/status', t('This action will classify all files that are missing a type, and may be a lengthy process. This action cannot be undone.'), t('Classify file types'), t('Cancel'));
```

Comment:
This confirm form is unnecessarily scary:
![image](https://user-images.githubusercontent.com/100206/192128229-612c0d14-4d20-48b7-aa7c-c1beee760973.png)

I think we can remove "This action cannot be undone", since it's non-destructive. And we can change the submit button to remove the `button-danger` class, so it shows as a primary (blue) button.

Reply:
I had to manually change the css type of the button, as `confirm_form` uses `button-danger` by default.